app/views/layouts/application.html.erb

```diff
@@ -101,18 +101,40 @@
   <%= render "govuk_publishing_components/components/layout_footer", {
     navigation: [
       {
-        title: "Support and feedback",
+        title: t("application.footer.support_and_feedback"),
         items: [
           {
-            href: "https://support.publishing.service.gov.uk/",
+            href: "https://signon.integration.publishing.service.gov.uk/users/sign_in",
```

Comment:
We should use [Plek](https://github.com/alphagov/plek) for this URL as it is integration environment specific.

Reply:
This also seems to be a sign in URL is this meant to be to a form?